Multisig/UI/Assets/BalancesViewModel.swift

```diff
@@ -48,6 +44,18 @@ struct TokenBalance: Identifiable, Hashable {
     }
 }
 
+extension TokenFormatter {
+
+    func safeString(from: String?, decimals: Int? = nil, isNegative: Bool = false) -> String {
+        string(
+            from: BigDecimal((isNegative ? -1 : 1) * (BigInt(from ?? "0") ?? 0), decimals ?? 18),
+            decimalSeparator: Locale.autoupdatingCurrent.decimalSeparator ?? ".",
+            thousandSeparator: Locale.autoupdatingCurrent.groupingSeparator ?? ",",
+            forcePlusSign: true
```

Comment:
please make this as another parameter to this function, with default value = true, but call it from the line 33 above with value of false